devops/openshift/database/redis-ha-deploy.yml

```diff
@@ -147,7 +147,7 @@ objects:
             resources:
               requests:
                 storage: ${PVC_SIZE}
-          storageClassName: ${STORAGE_CLASS}
+            storageClassName: ${STORAGE_CLASS}
```

Comment:
Just to support the change, `storageClassName` is defined under the specs as per the below doc.
https://docs.openshift.com/container-platform/4.13/rest_api/storage_apis/persistentvolumeclaim-v1.html#spec
Thanks for making the change.

Reply:
I redid a test on DEV and it worked both ways. 😄 That's good that we tested on TEST.

Reply:
I went back to the history now and checked that it did happen on DEV but I didn't pay much attention it was just a warning.
![image](https://github.com/bcgov/SIMS/assets/78114138/9c769744-0ebb-4100-bfab-f61e759d843d)